client/src/components/Dropdown/Dropdown.js

```diff
@@ -1,24 +1,51 @@
 import React, { Component } from 'react';
-import classnames from 'classnames';
+import classNames from 'classnames';
```

Comment:
Let's leave as `classnames` (lowercase) for consistency's sake with the rest of the project.